resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -0,0 +1,479 @@
+import {Link, useForm, usePage} from '@inertiajs/inertia-react'
+import AdapterDateFns from '@mui/lab/AdapterDateFns'
+import DatePicker from '@mui/lab/DatePicker'
+import LocalizationProvider from '@mui/lab/LocalizationProvider'
+import {
+    Box,
+    Button,
+    Grid,
+    Step,
+    StepLabel,
+    Stepper,
+    TextField,
+    Typography,
+} from '@mui/material'
+import Alert from '@mui/material/Alert'
```

Comment:
merge this with the import from `@mui/material` please, same for Chip, FormControl and all other components that can be imported from a single module